draft-ietf-rats-architecture.md

```diff
@@ -1310,8 +1310,8 @@ an isolated environment might be provided by a process, a dedicated chip,
 a TEE, a virtual machine, or another secure mode of operation.
 The Attesting Environment must be protected from unauthorized modification to ensure it behaves correctly. Confidentiality protection of the Attesting Environment's signing key is vital so it cannot be misused to forge Evidence.
 
-In many cases the user or owner of a device that takes on the role of Attester must not be able to modify or extract keys from its Attesting Environments.
-For example, the owner or user of a mobile phone or FIDO authenticator might not be trusted to use the keys to report Evidence about the environment that protects the keys.
+In many cases the user or owner of a device that includes the role of Attester must not be able to modify or extract keys via the Attesting Environments.
```

Comment:
"via the" quite significantly changes the meaning. The original text implies that the Attesting Environment relies on some secured key material. The new text now does something to keys of other environments via the Attesting Environment.

I am not saying that is not good content (although I don't seem to get it right now), but it removes existing content, I think.